content/en/docs/tasks/debug/debug-cluster/kubectl-node-debug.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Debugging Kubernetes nodes with kubectl
+content_type: task
+min-kubernetes-server-version: 1.20
+---
+
+<!-- overview -->
+This page shows how to debug a [node](/docs/concepts/architecture/nodes/) running on the Kubernetes cluster using `kubectl debug` command.
+
+## {{% heading "prerequisites" %}}
+
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+You need to have access to create Pods and to assign those new Pods to arbitrary nodes.
+You also need to be authorized to create Pods that access filesystems from the host.
+
+
+<!-- steps -->
+
+## Debugging a Node using `kubectl debug node`
+
+Use the `kubectl debug node` command to deploy a Pod to a Node that you want to troubleshoot.
+This command is helpful in scenarios where you can't access your Node by using an SSH connection.
+When the Pod is created, the Pod opens an interactive shell on the Node.
+To create an interactive shell on a Node named “mynode”, run:
+
+```shell
+kubectl debug node/mynode -it --image=ubuntu
+```
+
+```console
+Creating debugging pod node-debugger-mynode-pdx84 with container debugger on node mynode.
+If you don't see a command prompt, try pressing enter.
+root@ek8s:/#
+```
+
+Run debug commands to help you gather information and troubleshoot issues. Commands 
+that you might use to debug, such as `ip`, `ifconfig`, `nc`, `ping`, and `ps`. You can also
```

Comment:
Perhaps we need a note that commands will vary and may need to be installed
I tested this on a cluster and only `ping` and `ps` are available

Reply:
fixed